src/server/wasm/api.h

```diff
@@ -23,6 +23,7 @@ bool RegisterApiFunction(std::string_view name, Fn f, wasmtime::Linker* linker)
   auto args_signature = wasmtime::FuncType({wasmtime::ValKind::I32}, {});
 
   auto res = linker->func_new(module_name, name, args_signature, f);
+  linker->func_new("env", name, args_signature, f);
```

Comment:
yeah I need to find out how to tell wasi to look in a different namespace... I just couldn't! There is wasm-bindgen, but I don't wanna use it

Reply:
can be done with `wit` file https://benw.is/posts/compiling-rust-to-wasi